aea/agent.py

```diff
@@ -273,7 +273,10 @@ def get_message_handlers(self) -> List[Tuple[Callable[[Any], None], Callable]]:
 
         :return: List of tuples of callables: handler and coroutine to get a message
         """
-        return [(self.handle_envelope, self.inbox.async_get)]
+        return [
+            (self.handle_envelope, self.inbox.async_get),
+            (self.handle_envelope, self.runtime.agent_loop.skill2skill_queue.get),
```

Comment:
Should be added on aea level in same method. Agent should not know about skill2skill_queue

Reply:
done